dense_vector/challenges/default.json

```diff
@@ -23,10 +23,52 @@
       }
     },
     {
-      "operation": "index-append",
+      "name": "index-append",
+      "operation": {
+        "operation-type": "bulk",
+        "bulk-size": {{bulk_size | default(5000)}},
+        "ingest-percentage": 10
+      },
       "warmup-time-period": {{ bulk_warmup | default(40) | int }},
       "clients": {{bulk_indexing_clients | default(1)}}
     },
+    {
+      "parallel": {
+        "tasks": [
+          {
+            "name": "index-append-concurrent-with-searches",
+            "operation": {
+              "operation-type": "bulk",
+              "bulk-size": {{bulk_size | default(5000)}},
+              "ingest-percentage": 10
```

Comment:
I did this same thing in #195, and it turns out this will ingest the same 10% as the `index-append` task. FYI in case you need to write net-new docs for this challenge

Reply:
@DJRickyB Thanks for your comment. I did not know that. We want to index net-new documents in this part. What's they way to tell to index the next 10%?

Reply:
There is not one, unfortunately. One (maybe overly simplistic way) to accomplish some amount of data density before you start "counting" query performance would be to take out the initial `index-append` task and then give every task in the parallel block the same `warmup-time-period`. This would give the queries a minimum data density before the *measured* iterations, but would drop the in-a-vacuum indexing measurement, and you'd have to get that from running a separate challenge. I understand this approach is a bit sloppy, maybe we can chat out-of-band how to accomplish what you need or else enhance Rally to support what you're looking for here